src/components/adaptive-cards/Spacer/Spacer.scss

```diff
@@ -0,0 +1,52 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-ac-spacer;
+
+.#{$C} {
+  flex-shrink: 0;
+  &:first-child { display: none; }
+
+  .wxc-ac-container--vertical > & {
```

Comment:
how about `.wxc-ac-container--vertical > :not(:first-child)&`? does it work?

Reply:
No, this variant does not work.